Configuration/Skimming/python/RAWSkims_cff.py

```diff
@@ -0,0 +1,9 @@
+import FWCore.ParameterSet.Config as cms
+import HLTrigger.HLTfilters.hltHighLevel_cfi as hlt
+
+ReserveDMu = hlt.hltHighLevel.clone()
+ReserveDMu.TriggerResultsTag = cms.InputTag( "TriggerResults", "", "HLT" )
+ReserveDMu.eventSetupPathsLabel = cms.string('SecondaryDatasetTrigger')
+ReserveDMu.eventSetupPathsKey = cms.string('ReserveDMu')
+ReserveDMu.andOr = cms.bool( True )
+ReserveDMu.throw = cms.bool( False )
```

Comment:
```suggestion
ReserveDMu = hlt.hltHighLevel.clone()
ReserveDMu.TriggerResultsTag = ("TriggerResults", "", "HLT" )
ReserveDMu.eventSetupPathsLabel = 'SecondaryDatasetTrigger'
ReserveDMu.eventSetupPathsKey = 'ReserveDMu'
ReserveDMu.andOr = True
ReserveDMu.throw = False
```
or
```suggestion
ReserveDMu = hlt.hltHighLevel.clone(
   TriggerResultsTag = ("TriggerResults", "", "HLT" ),
   eventSetupPathsLabel = 'SecondaryDatasetTrigger',
   eventSetupPathsKey = 'ReserveDMu',
   andOr = True,
   throw = False,
)
```
is safer with respect to possible typos (e.g.)

Reply:
Thank you for the comment @perrotta